library/src/scss/userContent/_lists.scss

```diff
@@ -47,7 +47,14 @@ $nestedList-margin: 0 0 0 1em;
 .userContent {
     ul {
         li {
-            list-style: disc !important;
+            list-style: none !important;
+            position: "relative";
+
+            &::before {
+                content: `"•"`,
```

Comment:
Make sure to specify the font to something safe, like "arial" so it doesn't mess up if a bad font is selected.

Reply:
Done